detection_rules/etc/version.lock.json

```diff
@@ -3170,15 +3184,15 @@
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "Windows Event Logs Cleared",
-        "sha256": "7485e3272dcc60566ca499afce5cf1f87ab84c039d427a4ed6a522fd0a7d1bc0",
+        "sha256": "22523a171ded4e5880a944e7f2bd14015c141eb0f2a9fdea86bfe18ab758ecf7",
```

Comment:
Rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.

Reply:
Change resulted from manual ATT&CK updates pushed to these branches. Since all rules are min-stacked, but the ATT&CK mappings backport entirely, changes that are necessary to pass unit testing must occur directly on that branch.

Reference: https://github.com/elastic/detection-rules/pull/2422